l3kernel/l3basics.dtx

```diff
@@ -1687,7 +1687,7 @@
 %     \use_i:nnnnnnn, \use_ii:nnnnnnn, \use_iii:nnnnnnn, \use_iv:nnnnnnn ,
 %       \use_v:nnnnnnn , \use_vi:nnnnnnn , \use_vii:nnnnnnn ,
 %     \use_i:nnnnnnnn, \use_ii:nnnnnnnn, \use_iii:nnnnnnnn, \use_iv:nnnnnnnn ,
-%       \use_v:nnnnnnnn , \use_vi:nnnnnnnn , \use_vii:nnnnnnnn , \use_vii:nnnnnnnn ,
+%       \use_v:nnnnnnnn , \use_vi:nnnnnnnn , \use_vii:nnnnnnnn ,
```

Comment:
Oops, I was wrong.

It's not that `\use_vii:nnnnnnnn` was doubly listed, but `\use_viii:nnnnnnnn` (8th of 8) was mistakenly documented as `\use_vii:nnnnnnnn` (7th of 8). (Too many `n`s.)

```suggestion
%       \use_v:nnnnnnnn , \use_vi:nnnnnnnn , \use_vii:nnnnnnnn , \use_viii:nnnnnnnn ,
```

Line 969 needs a similar `vii` -> `viii` fix.